modules/blake3/1.8.2/overlay/c/BUILD.bazel

```diff
@@ -0,0 +1,179 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+package(default_visibility = ["//visibility:private"])
+
+config_setting(
+    name = "tbb_enabled",
+    define_values = {"blake3:tbb": "enabled"},
```

Comment:
New `--define`s should be avoided. Could you use a Starlark build setting (`bool_flag` from `bazel_skylib`) instead?